client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx

```diff
@@ -66,12 +66,30 @@ const ContactQuestioningPersonal: React.FC<Props> = (props: Props): JSX.Element
     const watchIdentificationType = watch(identificationTypeFieldName);
     const watchIdentificationNumber = watch(identificationNumberFieldName);
 
+    contactQuestioningService.getDuplicateIdentities().subscribe((duplicates) => {
+        const isDuplicateIdentity = duplicates.filter(obj => obj.identityType === methods.getValues(identificationTypeFieldName) && obj.identityNumber === methods.getValues(identificationNumberFieldName)).length !== 0;
+        setDuplicateIdentityValidation(isDuplicateIdentity);
```

Comment:
please change line 71 to this:
setIsDuplicateId(isDuplicateIdentity)
and remove lines 73-81